src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -309,6 +309,14 @@ internal static bool IsFileInSolution(ProjectItem projectItem)
             return false;
         }
 
+        private static Request ToRequest(FileConfig fileConfig, string path)
```

Comment:
Moved from `FileConfig` and made into a static method. I also changed the scope from `public` to `private`.